packages/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,34 @@
+const knex = require('../../config/db');
+
+// get movies that added in act by userID
+const getOrderIDCreatedStatus = (userID) => {
+  return knex('orders')
+    .select('*')
+    .where({ user_id: userID, status: 'created' });
+};
+
+// issue new order id in case, there is no order_id with status created
+let createdID = 0;
+const addMovieToCartWithNewOrderID = async (body) => {
+  const orderID = await knex('orders').insert({
+    user_id: body.user_id,
+    status: body.status,
+  });
+
+  await knex('order_items').insert({
+    order_id: orderID,
+    movie_id: body.movie_id,
+  });
+
+  createdID += 1;
```

Comment:
Same as above

Reply:
I also fixed this code too :) 
For orders table, I return orders.id itself, since this orders.id in this table set to auto_increment